core/development/logic/pieces/Unicorn.ts

```diff
@@ -10,6 +11,42 @@ export class Unicorn extends Piece {
     this.price = 4;
   }
 
+  getAttackablePieces(): Array<Piece> {
+    if (!this.position) return [];
+
+    const currentCoordinates = this.position.coordinates;
+    const attackablePieces = [];
+
+    // Offsets for the positions the unicorn can attack
+    const attackPositionsOffsets = [
+      { deltaX: 1, deltaY: 0 },
+      { deltaX: 0, deltaY: 1 },
+      { deltaX: 1, deltaY: 1 },
+      { deltaX: -1, deltaY: 0 },
+      { deltaX: 0, deltaY: -1 },
+      { deltaX: -1, deltaY: -1 },
+      { deltaX: -1, deltaY: 1 },
+      { deltaX: 1, deltaY: -1},
+    ]
```

Comment:
If you have to explain the code using a comment, it's not readable enough

Reply:
I was being consistent with the code below in the Unicorn class
``
    // Define all possible unicorn move offsets
    const unicornMoveOffsets = [
      { deltaX: 2, deltaY: 1 },
      { deltaX: 2, deltaY: -1 },
      { deltaX: -2, deltaY: 1 },
      { deltaX: -2, deltaY: -1 },
      { deltaX: 1, deltaY: 2 },
      { deltaX: 1, deltaY: -2 },
      { deltaX: -1, deltaY: 2 },
      { deltaX: -1, deltaY: -2 },
    ];
``

Reply:
I hate github
![image](https://github.com/Ido-Barnea/Chess-But-Better/assets/62211288/8f6e51f9-d866-4999-83f6-44153ae9fc72)

Reply:
I never said I wrote it well in the first place 😆 😢

Reply:
Should I just remove the comment? I think the name pretty much explains it

Reply:
sure